gradle/error-prone.gradle

```diff
@@ -13,21 +13,10 @@ tasks.withType(JavaCompile).configureEach {
 				'FormatStringConcatenation',
 				'FunctionalInterfaceMethodChanged',
 				'JavaxInjectOnAbstractMethod',
+				'MissingSummary',
 				'OverridesJavaxInjectableMethod',
 				'ReturnValueIgnored',
 				'Slf4jLogStatement',
-				'MissingSummary',
-				// patchable
-				'LexicographicalAnnotationAttributeListing',
```

Comment:
now its working as we have enabled convention. the warnings does not break the build somehow.